pkgs/top-level/python-packages.nix

```diff
@@ -1656,7 +1656,9 @@ in {
 
   datasette = callPackage ../development/python-modules/datasette { };
 
-  datashader = callPackage ../development/python-modules/datashader { };
+  datashader = callPackage ../development/python-modules/datashader {
+    dask = self.dask.override { withExtraComplete = true; };
```

Comment:
This change needs to be undone. This will lead to closures with the same package twice but slightly different.